seedfarmer/__main__.py

```diff
@@ -348,14 +356,23 @@ def list_module_metadata(
     deployment: str,
     group: str,
     module: str,
+    export_local_env: str,
     debug: bool,
 ) -> None:
     if debug:
         enable_debug(format=DEBUG_LOGGING_FORMAT)
-    _logger.debug(f"We are getting module data for {module} in {group} of {deployment}")
+    if _logger.isEnabledFor(logging.DEBUG):
+        _logger.debug("We are getting module data for  %s in %s of %s", module, group, deployment)
 
-    val = json.dumps(mi.get_module_metadata(deployment, group, module))
-    sys.stdout.write(val)
+    metadata_json = mi.get_module_metadata(deployment, group, module)
+    if not export_local_env:
+        sys.stdout.write(json.dumps(metadata_json))
```

Comment:
why the explicit write to stdout here and below? should these be log messages?

Reply:
the CLI support for metadata has always been to write to std out (the deployspec uses the output to export to env).

Should we change?

Reply:
got it. all good,